src/services/notifications/index.ts

```diff
@@ -520,3 +541,68 @@ export const NotificationsService = (logger: Logger): INotificationsService => {
     }),
   }
 }
+
+export const getPushNotificationContent = ({
```

Comment:
Could we also sketch out some different scenarios for this function in unit tests? I'm think for e.g.:
- sat base; usd display
- usd base; usd display
- sat base; crc display
- usd base; crc display

Reply:
Also, `get` so far in our codebase implies that this method is reaching out to somewhere external to retrieve something but I believe this is only doing local manipulation? If it's only local only:
- maybe we can use another prefix like `calculate` or `create` (I'm not sure what makes most sense here)
- we can maybe move this into notifications domain to be unit tested

Reply:
done here https://github.com/GaloyMoney/galoy/pull/1338/commits/ece539b4f02c5123900446e9406bd6966963f563 

We should not move this to domain because this is only related to the service and it is not used in app layer